executor/distsql.go

```diff
@@ -910,6 +957,75 @@ func (e *IndexLookUpExecutor) getHandle(row chunk.Row, handleIdx []int,
 	return
 }
 
+func recordIndexLookUpRuntimeStats(stats *tikv.RPCRuntimeStats, d time.Duration) {
+	atomic.AddInt64(&stats.Count, int64(1))
+	atomic.AddInt64(&stats.Consume, int64(d))
+}
+
+type indexLookUpRunTimeStats struct {
+	*tikv.SnapshotRuntimeStats
+	indexTask    *tikv.RPCRuntimeStats
+	tableTask    *tikv.RPCRuntimeStats
+	indexScan    int64
+	tableRowScan int64
+}
+
+func (e *indexLookUpRunTimeStats) String() string {
+	var buf bytes.Buffer
+	if e.indexScan != 0 && e.indexTask.Count != 0 {
+		buf.WriteString(fmt.Sprintf("index_task:{time:%s, num_rpc:%d}", time.Duration(e.indexScan), e.indexTask.Count))
```

Comment:
Normally, the `num_rpc` of index_task always be 1, so it is useless, how about remove it.

Reply:
if so, I think we can use `index_task_time:2s....` instead of` index_task{time:2s}`. what do you think

Reply:
great